src/main/java/org/jenkins/plugins/statistics/gatherer/listeners/QueueStatsListener.java

```diff
@@ -22,8 +22,8 @@
  */
 @Extension(dynamicLoadable = YesNoMaybe.YES)
 public class QueueStatsListener extends QueueListener {
-    private static final Logger LOGGER = Logger.getLogger(
-            QueueStatsListener.class.getName());
+
```

Comment:
With this and other formatting changes, it would be best to define a common syntax style for the whole project and stick with it.
What about GJF? (https://github.com/google/google-java-format)